be/src/storage/lake/delta_writer.cpp

```diff
@@ -248,7 +248,7 @@ inline Status DeltaWriterImpl::flush_async() {
     if (_mem_table != nullptr) {
         RETURN_IF_ERROR(_mem_table->finalize());
         if (_miss_auto_increment_column && _mem_table->get_result_chunk() != nullptr) {
-            _fill_auto_increment_id(*_mem_table->get_result_chunk());
+            RETURN_IF_ERROR(_fill_auto_increment_id(*_mem_table->get_result_chunk()));
```

Comment:
If `_fill_auto_increment_id` fail, will it affect `flush_async` ?.

Reply:
I think it will affect it. And I think it is the correct behavior if _fill_auto_increment_id fail.